doctor/tasks.py

```diff
@@ -189,43 +190,39 @@ def get_page_count(path, extension):
 
 def extract_from_pdf(
     path: str,
+    page_count: int,
     ocr_available: bool = False,
+    strip_margin: bool = False,
 ) -> Any:
     """Extract text from pdfs.
 
-    Start with pdftotext. If we we enabled OCR - and the the content is empty
-    or the PDF contains images, use tesseract. This pattern occurs because PDFs
-    can be images, text-based and a mix of the two. We check for images to
-    make sure we do OCR on mix-type PDFs.
-
     If a text-based PDF we fix corrupt PDFs from ca9.
 
+    Extract the text from the document and if ocr available OCR the document
+
     :param path: The path to the PDF
     :param ocr_available: Whether we should do OCR stuff
+    :param strip_margin: Whether to remove 1 inch margin from text extraction
     :return Tuple of the content itself and any errors we received
     """
-    content, err, returncode = make_pdftotext_process(path)
```

Comment:
If this is no longer used, we should remove the function and the dependency, right?

Reply:
indeed.